pkg/clients/has/components.go

```diff
@@ -182,6 +182,58 @@ func (h *HasController) WaitForComponentPipelineToBeFinished(component *appservi
 	return nil
 }
 
+// Universal method to create a component in the kubernetes clusters and adds a suffix to the component name to allow multiple components with unique names.
+// Generate a random component name with #combinations > 11M, Create unique resource names that adhere to RFC 1123 Label Names
+// https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
+func (h *HasController) CreateComponentV2(componentSpec appservice.ComponentSpec, namespace string, outputContainerImage string, secret string, applicationName string, skipInitialChecks bool, annotations map[string]string) (*appservice.Component, string, error) {
```

Comment:
This function mostly copies `CreateComponent`. Would it be possible to make just athin wrapper around `CreateComponent`?

Do you have an example of how CDQ for repo with multiple components looks like? Do we need to generate our own component name?

Reply:
Fixed duplication with CreateComponent and CreateComponentV2 functions